drivers/dai/intel/ssp/ssp.c

```diff
@@ -1759,15 +1785,25 @@ static void dai_ssp_set_reg_config(struct dai_intel_ssp *dp, const struct dai_co
 				   const struct dai_intel_ipc4_ssp_config *regs)
 {
 	struct dai_intel_ssp_pdata *ssp = dai_get_drvdata(dp);
-	uint32_t ssc0, sstsa, ssrsa;
+	uint32_t ssc0, sstsa, ssrsa, sscr1;
 
 	ssc0 = regs->ssc0;
-	sstsa = regs->sstsa;
-	ssrsa = regs->ssrsa;
+	sstsa = SSTSA_GET(regs->sstsa);
+	ssrsa = SSRSA_GET(regs->ssrsa);
+	sscr1 = regs->ssc1 & ~(SSCR1_RSRE | SSCR1_TSRE);
+
+	if (regs->sstsa & SSTSA_TXEN || regs->ssrsa & SSRSA_RXEN ||
+	    regs->ssc1 & (SSCR1_RSRE | SSCR1_TSRE)) {
+		LOG_INF("%s: Ignoring %s%s%s%sfrom blob", __func__,
```

Comment:
@ujfalusi we're seeing this error message pretty much everywhere:
- https://github.com/thesofproject/sof/issues/9026

Any idea why?

Reply:
Sorry the LOG_ERR was in commit 6423bc3bc885 actually.

Reply:
@marc-hb, no, this print remained LOG_INF:
https://github.com/zephyrproject-rtos/zephyr/commit/6423bc3bc8853997e6974cf2aa68c4c4ee2c3abe#diff-69c3bde06e9c145ab8321fcce947ac6d765aa2ffe873641125e6d85d41f45943L1797-R1770

The blob is incorrect, it has the TX/RX enabled and/or the TX/RX DMA enabled. This is invalid and it is ignored, we don't want to enable the SSP when we load the blob, we want to enable it when it needs to be enabled.
We can lower the level, but the blob should be fixed.

Reply:
> no, this print remained LOG_INF:

Yes that's what I meant, sorry for the confusion.

> We can lower the level, but the blob should be fixed.

So should the "Ignoring ..." line be a WARN maybe?

Reply:
I would leave it to INF, the blob might be coming from BIOS/ROM and not from tplg, it is hard to fix those.